builders/manage/sudo/minting.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Minting Native Tokens with Sudo 
+description: Learn how to use the Sudo key to manage your appchain and performing the privileged action of minting additional native tokens of your Tanssi appchain.
+---
+
+# Using Sudo to Mint Native Tokens
+
+## Introduction {: #introduction }
+
+[Sudo](https://paritytech.github.io/polkadot-sdk/master/pallet_sudo/index.html){target=\_blank} is a module that enables privileged runtime calls to be dispatched when called from the Sudo account. Sudo is sometimes colloquially referred to as a superuser or god-like account. This enables you to perform privileged actions in the course of managing your appchain, such as minting new native tokens. 
+
+In this guide, you'll learn how to use Sudo to properly mint new native tokens. This comprehensive guide shows how to check the balance of an existing account prior to assigning it a new balance with Sudo access. 
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+For the examples in this guide, you will need to have the following:
+
+ - A Tanssi Appchain (Snap or Dedicated)
+ - Your Appchain's Sudo account connected to your Appchain's Polkadot.js Apps. You can refer to the [Managing Sudo guide](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank} for instructions on injecting your Sudo account into Polkadot.js Apps
+
+If you're unsure what your Tanssi Appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network/){target=\_blank} underneath the **Properties** section.
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/sudo/minting/minting-1.webp)
+
+!!! note
+    You should always protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi Appchain.
+
+## Minting Tokens {: #minting-tokens }
+
+As you know, the Sudo account has the ability to perform privileged functions, including minting additional tokens. When setting up your Tanssi Appchain on [apps.tanssi.network](https://apps.tanssi.network/){target=\_blank}, you can specify genesis account balances. In other words, you have the ability to endow accounts with initial balances upon launching your Tanssi Appchain. However, you can also mint new tokens after launch with the help of the Sudo account.
```

Comment:
```suggestion
As you know, the Sudo account has the ability to perform privileged functions, including minting additional tokens. When setting up your Tanssi Appchain on the [Tanssi dApp](https://apps.tanssi.network/){target=\_blank}, you can specify genesis account balances. In other words, you have the ability to endow accounts with initial balances upon launching your Tanssi Appchain. However, you can also mint new tokens after launch with the help of the Sudo account.
```

this came back on a recent SEO report, we need to switch to descriptive link text as opposed to writing out the link itself